src/Workspaces/Remote/Core/Services/SolutionService.cs

```diff
@@ -146,14 +146,14 @@ private static Solution GetAvailableSolution(Checksum solutionChecksum)
             if (currentSolution?.Item1 == solutionChecksum)
             {
                 // asked about primary solution
-                return currentSolution.Item2;
+                return currentSolution?.Item2;
```

Comment:
@jasonmalinowski, if solutionChecksum is null, we may get the NullReferenceException, so I believe that this extra check is useful.